Tactics/SkipProof.lean

```diff
@@ -0,0 +1,42 @@
+import Lean
+
+open Lean Meta Elab Tactic
+
+namespace SkipProof
+
+register_option skip_proof.skip : Bool := {
+  defValue := False
+  descr := "If `skip` is true, `skip_proof p` skips running `p` and closes the goal with `skipProofAx`."
+}
+
+/--
+Surrounding a tactic with `skip_proof` will skip running the tactic and close the goal with `skipProofAx`.
+We only skip proofs `skip_proof.skip` is set to `true`,
+and we emit a warning to remind the user that proofs are being skipped.
+
+This is to be used, during development, with expensive tactics such as `omega`, `bv_omega`, `bv_decide`, `simp_mem`.
+This allows one to run `have expensive_hyp : ... := by omega`, check that this can be proven,
+and to then replace this with `have expensive_hyp : ... := by skip_proof omega`,
+which will skip the use of `omega`, and instead of `skipProoxAx`, ensuring Lean stays interactive.
```

Comment:
Wording went wonky here.. "and instead of `skipProoxAx`..."

Reply:
Fixed!